index.html

```diff
@@ -1,19 +1,89 @@
 <!doctype html>
 
-<html lang="en">
-<head>
-  <meta charset="utf-8">
+  <html lang="en">
 
-  <title>My Resume</title>
+<!--Head Start-->
+    <head>
 
-  <link rel="stylesheet" href="css/index.css">
+      <title>My Resume</title>
 
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
-</head>
+      <meta charset="utf-8">
+    
+      <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+      <link rel="stylesheet" href="css/index.css">
+    
+      <meta-name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
good job remember the viewport meta tag, not many people did. 

Also great job on bringing in the google fonts and hitting stretch, love it!